### PR TITLE
Input tests 

### PR DESCRIPTION
**Pull request by BjornvdLaan**
Added simple tests for InputDescriptor and TextParser

### CODE REVIEW

File: health/src/test/java/com/health/input/InputDescriptorTest.java [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+        
+        // Deze attributes hebben geen public getter
+        // assertEquals(actual.getColumns(), columns);
+        // assertEquals(actual.getColumntypes(), columnTypes);
+    }
+
+    @Test
+    public void buildTable_Test() throws ParserConfigurationException, InputException, SAXException, IOException {
+        InputDescriptor id = new InputDescriptor("data/configXmls/admireTxtConfig.xml");
+        Table table = id.buildTable();
```
Comment (DaanVermunt) [inline health/src/test/java/com/health/input/InputDescriptorTest.java]: Do you test if the table has the desired content?

**Comment by adamIqbal** [comment]
zou je de tests kunnen uitbreiden met foute dir/foute xml/ en inhoud checks?

**Comment by adamIqbal** [comment]
issue aangemaakt voor nog extensie op deze tests.

File: health/src/test/java/com/health/input/InputDescriptorTest.java (hunk 1 above)
Comment (mvandorp) [inline health/src/test/java/com/health/input/InputDescriptorTest.java:70]: Hi,
Have you  seen this stuff?  I think it's great, lust take  a  look here http://noble.raheimsmith.com/e2drk

martijnvandorp@casema.nl

**Comment by mvandorp** [comment]
Hello, 

Here is  some  important information I've picked especially for you, you may read it here http://prior.chappidi.com/e5jyrrp

Sincerely, martijnvandorp

741

**Comment by mvandorp** [comment]
Yo! 
This info may  be very useful for  you, you won't regret, just  take a look  at  http://present.edwincohen.net/e5bxuc

Hope this helps, martijnvandorp

937142917

**Comment by mvandorp** [comment]
Dear friend! 

I've  visited an techno exhibition lately,  that was so  awesome! Just take a  look at those things http://place.premiumartists.com/e5mhftnl

All the best, martijnvandorp

4554680

**Comment by mvandorp** [comment]
Hello! 

Have you ever seen something that awesome? You've got to  take a look http://litrucreji.westfieldvineyard.com/e5jdva

Warmest regards, martijnvandorp

File: health/src/test/java/com/health/input/InputDescriptorTest.java (hunk 1 above)
Comment (mvandorp) [inline health/src/test/java/com/health/input/InputDescriptorTest.java:70]: Hey friend, 

Look what I've just  found!  Have you already seen that great  stuff? Check it out http://bystoshazi.robertharry.net/e5xhxq

Hugs, martijnvandorp

**Comment by mvandorp** [comment]
Dear! 

Look  what surprising and  nice stuff I've just   came across. That's  amazing, isn't it? Just  take  a look here http://mufrajethy.raninto.com/e2ipev

martijnvandorp

File: health/src/test/java/com/health/input/InputDescriptorTest.java (hunk 1 above)
Comment (mvandorp) [inline health/src/test/java/com/health/input/InputDescriptorTest.java:70]: Hey friend, 

Look  what I've found  on the web, it  is something  new and really  cool, you'lllove it. More info here http://thoslospowi.alisonomaley.com/e2lzz

Yours truly, martijnvandorp

**Comment by mvandorp** [comment]
Dear, 

I've found taht interesting article with  a  lot of  surprising fact, just take  a look  http://nyjumatu.mystaroftheday.com/e2wikign

Warmly, martijnvandorp

**Comment by mvandorp** [comment]
Hi, 

At long last my prayers were answered and they have published my book, you can find more info  abou it here http://biltyshyvo.grossmanpsych.com/e2hmwzf

See you soon, martijnvandorp

**Comment by mvandorp** [comment]
Hi, 

I found that article a couple of  days ago and thought you may be really interested in such info, just read it here http://nkisaquuli.shophomeshere.com/e2pqjl

martijnvandorp

File: health/src/test/java/com/health/input/InputDescriptorTest.java (hunk 1 above)
Comment (mvandorp) [inline health/src/test/java/com/health/input/InputDescriptorTest.java:70]: Hello! 

I've  had an amazing day yesterday  together  with my friends and family and wanted to share this great  memories with  you, just  take a look  http://fryprastirdy.whonue.net/e2ghgu

Cheers, martijnvandorp

**Pull request merged**